src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/hellfire_ramparts.h

```diff
@@ -6,6 +6,9 @@
 #define DEF_RAMPARTS_H
 
 #include "SpellScript.h"
+#include "CreatureAIImpl.h"
+
+#define HellfireRampartsScriptName "instance_hellfire_ramparts"
```

Comment:
This is an error. I'm sorry I just noticed this.

Reply:
https://github.com/Kitzunu/azerothcore-wotlk/blob/058aea0652d40241704db91c5a177f63f4b59d32/src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/instance_hellfire_ramparts.cpp#L12

Reply:
I don't understand when you just link a random line.

I don't think this is an error as I just define the script name for the instance.
This means that if someone in the future decides to script the creatures in C++ they have this already made.

Reply:
The bio code script in the instance does not work

Reply:
Im sorry for not understanding.
Give me some steps to test and I can check tomorrow

Reply:
Enter the instance and fight with the boss. You will find that they are just like outside the instance. The script doesn't work

Reply:
Change the line to
```c++
 #define HellfireRampartsScriptName "instance_ramparts"
 ```
Fight again, and you'll find the script works

Reply:
Oh now I get it.
Tbh we should change it to instance_hellfire_ramparts in the scropt file.

I will do this tomorrow, good catch 👍

Reply:
```sql
UPDATE `instance_template` SET  script = 'instance_hellfire_ramparts' WHERE `map` = 543;
UPDATE `instance_template` SET  script = 'instance_utgarde_pinnacle' WHERE `map` = 575;
```